mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -15,8 +15,10 @@
 
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/EmitC/IR/EmitC.h"
+#include "mlir/Dialect/EmitC/Transforms/TypeConversions.h"
 #include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/BuiltinTypes.h"
+#include "mlir/IR/Region.h"
```

Comment:
I don't think this is needed?

Reply:
You're right, it is not. That's a remnant of the first implementation I had done of the shifts, that used an if/then/else instead of a ternary operator. I removed that. Thanks!